### PR TITLE
DOCSP-45086 IntelliJ Landing Page

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION 
- Landing page for the MongoDB for IntelliJ Plugin 

## STAGING 
https://deploy-preview-2--docs-mongodb-intellij.netlify.app/

## JIRA
https://jira.mongodb.org/browse/DOCSP-45086

### CODE REVIEW

File: source/index.txt
Comment (jmd-mongo) [inline source/index.txt:20]: [minor question]

I'm curious if we need to mention that this tool builds on a pre-existing one. I think we might be able to omit this and retain the same meaning,unless there's something I'm missing. My thinking here is that these opening sentences should be as succinct as possible.

File: source/index.txt
Comment (jmd-mongo) [inline source/index.txt:33]: [praise]

The overall structure and flow is great.

File: source/index.txt
Comment (jmd-mongo) [inline source/index.txt]: [minor suggestion]

Consider shortening these introductory lines a bit. 

You might consider: "The {+intellij-full+} is designed to improve...". You could even consider: "The {+intellij-full+} improves the development experience..." though perhaps that's a bit too strong.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for these updates, @ajhuh-mdb! I've left a few small comments for your consideration. This is a great start.

Thanks.
Joe

File: source/index.txt
Comment (ajhuh-mdb) [reply source/index.txt:20]: I can double check with Gaurab, but AFAIK I intentionally included that it's building on a pre-existing tool since it's a UI element that IntelliJ users are already familiar with, which now will include MDB-specific warnings and feature compatibility.

File: source/index.txt
Comment (ajhuh-mdb) [reply source/index.txt:33]: yay thank you!

File: source/index.txt
Comment (ajhuh-mdb) [reply source/index.txt]: Can double check with Gaurab on this! Wanted to be a bit more conservative with this since the product _is_ still in development, so the latter suggestion could be a bit too strong.

File: source/index.txt
Comment (ajhuh-mdb) [reply source/index.txt:20]: Also, both the [product description](https://docs.google.com/document/d/1QThlSEo-PVI9ck-M3vjcAj-RPGDe3Gu_i3U5ZxyY8UY/edit?tab=t.0#heading=h.o8zrnxlz4kcx) and [drafted private preview docs](https://docs.google.com/document/d/1aO9Cap2eX7CttYXEeWqmi-vy-XBLef5VNhegQGKTHzI/edit?tab=t.0#heading=h.8w6fsoolxwt6) highlight this aspect of expanding on the existing tool, so I figured it warranted a mention in the docs!

File: source/index.txt
Comment (ajhuh-mdb) [reply source/index.txt]: But opted to shorten like your first suggestion!

**Comment by ajhuh-mdb** [comment]
Hi @jmd-mongo 👋 Thanks so much for the review! This should be ready for another pass whenever you get the chance.

File: source/index.txt
Comment (jmd-mongo) [reply source/index.txt:20]: Sounds good -- thanks for clarifying!

**Review by jmd-mongo — APPROVED** [review]
Thanks for your work on this, @ajhuh-mdb! This one LGTM.

Thanks,
Joe

File: source/index.txt
Comment (GaurabAryal) [inline source/index.txt]: Suggestion:
Write, test, and iterate Java queries faster. Supports MongoDB Java Driver and Spring Data MongoDB.

File: source/index.txt
Comment (GaurabAryal) [inline source/index.txt]: did we mean to put https://jira.mongodb.org/projects/INTELLIJ/issues/ instead?

**Review by GaurabAryal — APPROVED** [review]
This is really great! Added a couple of suggestions but feel free to address & merge without a second review, thanks Ali :-).

File: source/index.txt
Comment (ajhuh-mdb) [reply source/index.txt]: Ah good catch. Fixed.

File: source/index.txt
Comment (ajhuh-mdb) [reply source/index.txt]: I'll add the note on support for the java driver & spring data in the first paragraph!

**Pull request merged**
